src/main/kotlin/AppInput.kt

```diff
@@ -0,0 +1,21 @@
+import java.util.Scanner
+
+class AppInput {
```

Comment:
Это можно сделать объектом, чтобы функции можно было вызывать без создания экземпляра этого класса:
```suggestion
object AppInput {
```